packages/commonwealth/client/scripts/views/components/Breadcrumbs/data.ts

```diff
@@ -0,0 +1,108 @@
+export const breadCrumbURLS = [
+  {
+    url: 'dashboard/for-you',
+    breadcrumb: 'Dashboard',
+  },
+  {
+    url: 'dashboard/global',
+    breadcrumb: 'Dashboard',
+  },
+  {
+    url: 'dashboard/chain-events',
+    breadcrumb: 'Dashboard',
+  },
+  {
+    url: 'notification-settings',
+    breadcrumb: 'Notification Management',
+  },
+  {
+    url: 'profile/id',
+    breadcrumb: 'Profile',
+  },
+  {
+    url: 'profile/edit',
+    breadcrumb: 'Edit',
+  },
+  {
+    url: 'createCommunity/starter',
+    breadcrumb: 'Create Community',
+  },
+  {
+    url: 'manage',
+    breadcrumb: 'Manage',
+    isAdmin: true,
+  },
+  {
+    url: 'analytics',
+    breadcrumb: 'Analytics',
+    isAdmin: true,
+  },
+  {
+    url: 'overview',
+    breadcrumb: 'Overview',
+  },
+  {
+    url: 'dicussions/General',
+    breadcrumb: 'Discussions/General',
+  },
+  {
+    url: 'dicussions/Proposal%20Discussion',
+    breadcrumb: 'Discussions/Proposal Discussion',
```

Comment:
the topics cannot be hardcoded because those are dynamic and differ between communities

Reply:
Spoke about this async but for public knowledge I was doing some local testing and it made it into my commit.